frontend/gateway/grpcclient/client_windows.go

```diff
@@ -0,0 +1,25 @@
+//go:build windows
+
+package grpcclient
+
+import (
+	"context"
+	"net"
+	"os"
+
+	"github.com/Microsoft/go-winio"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+func nPipeDialer() (string, grpc.DialOption) {
+	addr := os.Getenv("FrontendGRPCPipe")
```

Comment:
Referencing [this comment](https://github.com/moby/buildkit/pull/5868/files#r2056169169), `os.Getenv(BUILDKIT_SESSION_ID)` should work here to reconstruct the named pipe.